### PR TITLE
HBW-175 Add short tasks list for popup

### PR DESCRIPTION
**Pull request by TimAle**
Снимок экрана 2019-10-08 в 18 41 45

### CODE REVIEW

File: hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx [hunk 1]
```diff
@@ -54,14 +54,14 @@ const HBWClaimingTaskOverview = ({
...
         {!assigned && (
-          <button onClick={claimTask}
-                  className='claim-button'>{t('components.claiming.overview.claim_task')}
-          </button>
-        )}
-        {!assigned && (
-          <button onClick={claimAndGo}
-                  className='claim-button'>{t('components.claiming.overview.claim_and_open')}
-          </button>
+          <>
```
Comment (TimAle) [inline hbw/app/javascript/packs/hbw/components/claiming/task_overview.jsx]: Now we have fragments! 🎉

File: hbw/app/javascript/packs/hbw/widget.js.jsx [hunk 2]
```diff
@@ -44,7 +44,8 @@ modulejs.define(
           entity_code:      this.options.entity_code,
           initialVariables: payload.variables,
           fetch_all:        this.options.fetch_all,
-          poll_interval:    this.options.poll_interval
+          poll_interval:    this.options.poll_interval,
+          taskListPath:     this.options.taskListPath,
```
Comment (sugarfree1) [inline hbw/app/javascript/packs/hbw/widget.js.jsx]: Why do we mix camelCase with snake_case?

Comment (TimAle) [reply hbw/app/javascript/packs/hbw/widget.js.jsx]: Bad thing, will fix it later.

File: hbw/app/javascript/packs/hbw/components/claiming/short_list.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  const viewAll = () => { window.location.href = `${connection.options.host}/${taskListPath}`; };
+  const goToTask = (entityUrl) => { window.location.href = entityUrl; };
+
+  return (
+    <div className="short-list">
+      {tasks.map(({
+        id, icon, name, due, created, entityUrl
+      }) => (
+        <div onClick={() => goToTask(entityUrl)} key={id} className="row">
```
Comment (sugarfree1) [inline hbw/app/javascript/packs/hbw/components/claiming/short_list.jsx]: We need to color line with expired task

**Comment by TimAle** [comment]
UPD

**Pull request merged**